src/Symfony/Component/Serializer/Debug/TraceableNormalizer.php

```diff
@@ -31,6 +31,7 @@ class TraceableNormalizer implements NormalizerInterface, DenormalizerInterface,
     public function __construct(
         private NormalizerInterface|DenormalizerInterface $normalizer,
         private SerializerDataCollector $dataCollector,
+        private readonly string $serializerName,
```

Comment:
Adding a new required constructor argument is a BC break

Reply:
Fixed, I add a default value.